certified-connectors/absentify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,496 @@
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "Request requester custom id"
+                    }
+                  },
+                  "description": "Request requester"
+                },
+                "canceld_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "Request canceled by email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "Request canceled by name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "Request canceled by id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "Request canceled by custom id"
+                    }
+                  },
+                  "description": "Request canceled by"
+                },
+                "declined_by_member": {
+                  "type": "object",
+                  "properties": {
+                    "email": {
+                      "type": "string",
+                      "description": "Request declined by email"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "Request declined by name"
+                    },
+                    "id": {
+                      "type": "string",
+                      "description": "Request declined by id"
+                    },
+                    "custom_id": {
+                      "type": "string",
+                      "description": "Request declined by custom id"
+                    }
+                  },
+                  "description": "Request declined by"
+                },
+                "leave_type": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "Leave type id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "Leave type name"
+                    }
+                  },
+                  "description": "Leave type"
+                }
+              },
+              "description": "details"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Request created",
+        "description": "This trigger is executed when a new request is created in your company.",
+        "operationId": "trigger_request_created",

```

Comment:
@marc-on-github operationId properties should [use pascal case](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

Reply:
fixed